django/db/backends/base/schema.py

```diff
@@ -515,10 +519,10 @@ def _alter_field(self, model, old_field, new_field, old_type, new_type,
         if old_field.primary_key and new_field.primary_key and old_type != new_type:
             # '_meta.related_field' also contains M2M reverse fields, these
             # will be filtered out
-            for rel in _related_non_m2m_objects(new_field.model._meta):
-                rel_fk_names = self._constraint_names(rel.related_model, [rel.field.column], foreign_key=True)
+            for old_rel, new_rel in _related_non_m2m_objects(old_field, new_field):
```

Comment:
I would either rename `old_rel` to `_` or `_old_rel` here to denote it's unused.